pyairtable/api/abstract.py

```diff
@@ -16,25 +16,31 @@
 
 class ApiAbstract(metaclass=abc.ABCMeta):
     VERSION = "v0"
-    API_BASE_URL = "https://api.airtable.com/"
     API_LIMIT = 1.0 / 5  # 5 per second
-    API_URL = posixpath.join(API_BASE_URL, VERSION)
     MAX_RECORDS_PER_REQUEST = 10
+    DEFAULT_API_BASE_URL = "https://api.airtable.com/"
```

Comment:
We probably don't need this as a class attribute at all, do we? It could just be the default param in `__init__`

Reply:
https://github.com/gtalarico/pyairtable/pull/243/commits/18a4070dfbbf880f9dcf004bac3b70be153035a9

https://github.com/gtalarico/pyairtable/pull/243/commits/5068bca7090758e967ccad7f617bee75c3668378